includes/service-providers/active_campaign/class-newspack-newsletters-active-campaign.php

```diff
@@ -792,4 +791,22 @@ public function update_contact_lists( $email, $lists_to_add = [], $lists_to_remo
 		}
 		return true;
 	}
+
+	/**
+	 * Get contact data by email.
+	 *
+	 * @param string $email_address Email address.
+	 *
+	 * @return array|WP_Error Response or error if contact was not found.
+	 */
+	public function existing_contact_data( $email_address ) {
```

Comment:
NIT: `contact_data` presumes it exists, no need to state that.

```suggestion
	public function contact_data( $email_address ) {
```

Reply:
Done in d4727588a9029075dec1dbc345c376759fe034fc (and in Newspack Plugin, too)